### PR TITLE
Update documentation with conclusions from VDD19Q3

### PR DESCRIPTION
**Pull request by hermunn**
This has been a collaboration with @nigoroll, squashed together for
your convenience.

Closes: #3071

### CODE REVIEW

File: doc/sphinx/dev-guide/index.rst [hunk 1]
```diff
@@ -6,21 +6,6 @@ The Varnish Developers Guide
...
-
-* If still in doubt, ask.
-
-* Admit your mistakes, it's faster that way.
-
-* Thou SHALL not paint `bikesheds. <http://bikeshed.org/>`_
-
-* We will toss you out of the project rather than add another rule.
-
 Technical stuff
```
Comment (bsdphk) [inline doc/sphinx/dev-guide/index.rst]: This cannot be removed until it is replaced with something else.

The last line in particular is important, both now and in whatever replaces this:  Trying to reign in sociopaths by circumscribing their actions post-factum is not a workable model.

File: doc/sphinx/dev-guide/index.rst [hunk 2]
```diff
@@ -41,22 +26,46 @@ Technical stuff
...
+Bugs, pull requests, feature requests
 -------------------------------------
 
-Bugs, issues and feature requests start out as github issues.
+Bugs, pull requests and feature requests start out as github issues.
 
-Monday at 13:00-14:00 (EU time) we "bug-wash" on IRC to
-decide who and how issues are dealt with.
+Monday at 15:00-15:30 (EU time) we "bug-wash" on IRC to decide on the
+who and how of issues to be dealt with.
```
Comment (nigoroll) [inline doc/sphinx/dev-guide/index.rst]: should be reworded, suggestions from irc:

> (19:43:32) phk: the previous version probably should have been "by whom and how..."
> (19:43:33) slink: then maybe more verbose: "decide who handles which issue and how"

File: doc/sphinx/dev-guide/policy_governance.rst [hunk 3]
```diff
@@ -1,28 +1,62 @@
...
-project, but does little to nothing on a daily basis.
-
-The important thing is that there is agreement beforehand about who
-has the authority to step in, if need be.
-
-Appointment to the VGB is by acclamation, and we prefer the members
-to have enough distance and neutrality to be able to resolve
-conflicts, should it ever become necessary.
-
-Current members of the VGB:
```
Comment (nigoroll) [inline doc/sphinx/dev-guide/policy_governance.rst]: > (19:44:04) phk: Also, I think it is a bad idea to erase the current VGB until a new one is appointed.
> (19:45:00) slink: yeah, I agree

Comment (nigoroll) [reply doc/sphinx/dev-guide/policy_governance.rst]: > (19:45:28) phk: I would leave the current VGB section there, and add the one about a new and more formal project footing beneath it

**Comment by bsdphk** [comment]
First, removing the "behaviour" section was not talked about at all at the VDD and until we have something to replace it, removing it is not OK.

Second, I think we should move all this from the docs to homepage, so that outdated versions do not get stuck as we cut releases.  If it lives on the home-page there is only one copy and it will always be current.  the docs can link to the homepage for reference.

Third, as @nigoroll pasted in above, I think it may be a good idea to go for a three-part approach, along the lines of "day-to-day" (behaviour, most of what's in index today), "current governance" (essentially current governance page) and "future governance" for the slink/martin mandate and long term efforts.

**Comment by hermunn** [comment]
I think moving to the homepage is a good idea. Then we can simply link to the homepage from all branches, and have an updated version there. Do you want me (in collusion with @nigoroll) to open a PR there? When that is merged, I can change this PR to something that simply points to the homepage.

**Comment by bsdphk** [comment]
@hermunn yeah, lets move this task to https://github.com/varnishcache/homepage

**Comment by nigoroll** [comment]
has long been moved to https://github.com/varnishcache/homepage/pull/80

**Pull request closed without merging**